salt/modules/mysql.py

```diff
@@ -1796,6 +1810,9 @@ def user_create(
     .. versionadded:: 0.16.2
         The ``allow_passwordless`` option was added.
 
+    comments
+        The comments to use for the new user.
```

Comment:
**chore (blocking):** please add a `versionadded` note for this new option

Reply:
What would be the version?

Reply:
realistically, it'll probably be labeled for 3007.0